modules/widgets/image-widget.php

```diff
@@ -70,7 +70,10 @@ public function widget( $args, $instance ) {
 
 		if ( '' != $instance['img_url'] ) {
 
-			$output = '<img src="' . esc_attr( $instance['img_url'] ) .'" ';
+			$output = '<img src="' . esc_url( jetpack_photon_url( $instance['img_url'], array(
```

Comment:
I added only `Jetpack::is_module_active( 'photon' )`. Note that in many widgets (like Follow Button) we call `Jetpack::is_active()` on `widgets_init` and only then the widget is registered so by the time of widget registration the `Jetpack` class (and its methods) exists.